cli/command/environment_test.go

```diff
@@ -338,19 +296,93 @@ func TestEnvironmentSetMinCount(t *testing.T) {
 	})
 }
 
-func TestEnvironmentLink(t *testing.T) {
+func TestEnvironmentLinkBiDirectional(t *testing.T) {
 	testWaitHelper(t, func(t *testing.T, wait bool) {
 		base, ctrl := newTestCommand(t)
 		defer ctrl.Finish()
 
 		command := NewEnvironmentCommand(base.Command())
 
-		job := &models.Job{
-			JobID:  "job_id",
+		links1 := []string{"env_id2"}
+		links2 := []string{"env_id3", "env_id1"}
+
+		base.Resolver.EXPECT().
+			Resolve("environment", "env_name1").
+			Return([]string{"env_id1"}, nil)
+
+		base.Resolver.EXPECT().
+			Resolve("environment", "env_name2").
+			Return([]string{"env_id2"}, nil)
+
+		base.Client.EXPECT().
+			ReadEnvironment("env_id1").
+			Return(&models.Environment{
+				EnvironmentID: "env_id1",
+				Links:         []string{},
+			}, nil)
+
+		base.Client.EXPECT().
+			ReadEnvironment("env_id2").
+			Return(&models.Environment{
```

Comment:
we typically break-out these objects into their own variables if they become multi-line. It's pretty hard to read in the current format.

Reply:
✅